chart/etcd-backup-restore/values.yaml

```diff
@@ -90,6 +90,8 @@ backup:
   #   accessKeySecret: secret-access-key-with-object-storage-privileges
   #   accessKeyID: access-key-id-with-object-storage-privileges
 
+metrics: basic
```

Comment:
Do we really need this? Please don't add it unless there is  some strong use case. I preferred not to expose etcd configuration unless there is use case(specific requirement) for allowing that field configurable here. That's why i ignored many flags from https://github.com/etcd-io/etcd/blob/master/Documentation/op-guide/configuration.md.

Reply:
This metric is exposed and set in the Gardener etcd chart at https://github.com/gardener/gardener/blob/17354845eeaae0461f38f95639301d77f6f16e69/charts/seed-controlplane/charts/etcd/values.yaml#L17. So I felt it's necessary to expose this in etcd-backup-restore chart as well.

Reply:
Cool 👍